tests/integration/cloud/clouds/test_vmware.py

```diff
@@ -88,7 +88,11 @@ def test_instance(self):
         profile_config = cloud_config(profile)
         disk_datastore = profile_config['vmware-test']['devices']['disk']['Hard disk 2']['datastore']
 
+        for key, value in profile_config.get('vmware-test', {}).items():
```

Comment:
is fine for a test, but typically it's better to use six.iteritems() instead of items()

Reply:
I'll make that change